pubspec.yaml

```diff
@@ -75,9 +75,10 @@ dev_dependencies:
 flutter_icons:
   android: true
   ios: true
-  image_path: "assets/graphics/static-logo.png"
+  image_path: "assets/graphics/static-logo-no-name.png"
+  image_path_android: "assets/graphics/static-logo-no-name-android.png"
   adaptive_icon_background: "#FDF7B9"
```

Comment:
This hex colour value needs to be updated to the new background colour.

Reply:
change to Black as the background color in Ios?

Reply:
You should use colours as set in the design doc. The iOS image you've used will also need to be changed.
You can find the colour palette [here](https://github.com/voteflux/voting_app/issues/142#issuecomment-691466612)
The darkest colour would be best (#1C1F27)

Reply:
Ok, I will modify it shortly.